site/en/getstarted/cluster/install_cluster-milvusoperator.md

```diff
@@ -7,27 +7,27 @@ group: install_cluster-docker.md
 summary: Learn how to install Milvus cluster on Kubernetes using Milvus Operator
 ---
 
-# Install Milvus Cluster
-
-{{fragments/installation_guide_cluster.md}}
-
 {{tab}}
 
-## Create a Kubernetes Cluster
+# Install Milvus Cluster with Milvus Operator
 
-If you have already deployed a K8s cluster for production, you can skip this step and proceed directly to [deploy Milvus Operator](install_cluster-milvusoperator.md#Deploy-Milvus-Operator). If not, you can follow the following steps to quickly create a K8s for testing, and then use it to install a Milvus cluster with Milvus Operator. This tutorial introduces two ways to create a Kubernetes cluster:
+Milvus Operator is a solution that helps you deploy and manage a full Milvus service stack to target K8s clusters. The stack includes all Milvus components and relevant dependencies like etcd, Pulsar and MinIO. This topic introduces how to deploy a Milvus cluster with Milvus Operator on Kubernetes.
 
-- Use minikube to create a Kubernetes cluster in a virtual machine (VM).
-- Use kind to create a Kubernetes cluster in docker.
+## Prerequisites
+- [Check the requirements for hardware and software](prerequisite-helm.md) prior to your installation.
 
-<div class="alert note">
-The K8s cluster created by minikube and kind are for testing only.  Do <b>not</b> use it in production.
-</div>
+## Create a Kubernetes Cluster
+
+If you have already deployed a K8s cluster for production, you can skip this step and proceed directly to [deploy Milvus Operator](install_cluster-milvusoperator.md#Deploy-Milvus-Operator). If not, you can follow the steps below to quickly create a K8s for testing, and then use it to install a Milvus cluster with Milvus Operator. 
 
 ### Create a K8s cluster with minikube
 
 [minikube](https://minikube.sigs.k8s.io/docs/) is a tool that allows you to run Kubernetes locally.
 
+<div class="alert note">
+minikube can only be used in test environments. It is not recommended that you deploy Milvus distributed clusters in this way in production environments.
+</div>
+
 #### 1. Install minikube
 
 See [Prerequisites](prerequisite-helm.md#Software-requirements) for more information.
```

Comment:
See [install minikube](https://minikube.sigs.k8s.io/docs/start/) for more information.

(The webpage in this link includes the prerequisites and instructions on how to install minikube. I think it will be more accurate than our own prerequisite section. What's more, "prerequisite" here is quite confusing at first sight, I am not sure whether it refers to the prerequisite section in this topic or another prerequisite doc.

Reply:
[3837497](https://github.com/milvus-io/milvus-docs/pull/1643/commits/3837497953ae0d6003d9fbafc4e6e250bc7f0e40)